src/EventsBundle/Model/Event.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace SWP\EventsBundle\Model;
+
+class Event
```

Comment:
interface is missing + I would move all models to the Event Component - we will have mess in that bundle with time (a lot of classes coupled to the bundle, while they can be placed in the component and re-used by other projects independently if needed + the code will be well structured)